File: BiteClub/Models/Location.swift
import Foundation

struct Location: Codable, Identifiable {
    var id: UUID
    var name: String
    var address: String
    
    var orderLocations: [OrderLocation]?
    
    enum CodingKeys: String, CodingKey {
        case id = "id"
        case name = "name"
        case address = "address"
        case orderLocations = "order_locations"
    }
}

File: BiteRunr/shared/src/commonMain/kotlin/org/biterunr_studios/biterunr/Models/Location.kt
package org.biterunr_studios.biterunr.Models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Location(
    @SerialName("id")
    val id: String,

    @SerialName("name")
    val name: String,

    @SerialName("address")
    val address: String,

    @SerialName("order_locations")
    val orderLocations: List<OrderLocation>? = null
)
